src/aere_repl.erl

```diff
@@ -477,6 +483,11 @@ get_ready_chain(_) ->
     throw({repl_error, aere_msg:chain_not_ready()}).
 
 set_option(Option, Args, S = #repl_state{options = Opts}) ->
+    Locked = maps:get(locked_opts, Opts, []),
+    case lists:member(Option, [locked_opts|Locked]) of
+        true -> throw({repl_error, aere_msg:locked_option()});
+        _ -> ok
+    end,
```

Comment:
This may look nicer:

```suggestion
    LockedOption = lists:member(Option, [locked_opts|Locked]),
    LockedOption andalso throw({repl_error, aere_msg:locked_option()}),
```